server/document.js

```diff
@@ -44,7 +44,7 @@ export class Head extends Component {
   }
 
   getCssLinks () {
-    const { assetPrefix, files } = this.context._documentProps
+    const { assetPrefix, assetCrossOrigin, files } = this.context._documentProps
```

Comment:
Can this be the same thing as `nonce` where it's read from `this.props` instead?

Reply:
This way it doesn't have to be passed all the way through render etc.

Reply:
But set on `NextScript` in _document.js

Reply:
@timneutkens It would then work something like this passing attribute to Head and NextScript?  
_document.js
```
<Head crossOrigin="anonymous">...</Head>
<body>
   <Main/>
   <NextScript crossOrigin="anonymous"/>
</body>
```
 
That implementation proposal does seem like it would give users the same capability with less code changes!  The disadvantage being that users would need to extend _document.js and pass the same desired value to do different places (Head/NextScript) but that seems reasonable to me.

Reply:
I created https://github.com/zeit/next.js/pull/5646 to close this out.  Please take a look.